<ui version="4.0" >
 <class>GrammarSettingsDlg</class>
 <widget class="QWidget" name="GrammarSettingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Grammatik Einstellungen</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="KEditListBox" name="elbSentences" >
     <property name="title" >
      <string>Satzstrukturen</string>
     </property>
     <property name="buttons" >
      <set>KEditListBox::Add|KEditListBox::Remove</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="KPushButton" name="pbImportTexts" >
       <property name="text" >
        <string>Textimport</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbRename" >
       <property name="text" >
        <string>Terminal umbenennen</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbMerge" >
       <property name="text" >
        <string>Terminale verbinden</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>KPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <connections/>
</ui>
